tgbotapi.core/src/commonMain/kotlin/dev/inmo/tgbotapi/types/gifts/AcceptedGiftTypes.kt

```diff
@@ -0,0 +1,21 @@
+package dev.inmo.tgbotapi.types.gifts
+
+import dev.inmo.tgbotapi.types.limitedGiftsField
+import dev.inmo.tgbotapi.types.premiumSubscriptionField
+import dev.inmo.tgbotapi.types.uniqueGiftsField
+import dev.inmo.tgbotapi.types.unlimitedGiftsField
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+
+@Serializable
+data class AcceptedGiftTypes(
```

Comment:
As I see, [Bot API](https://core.telegram.org/bots/api#AcceptedGiftTypes) declare these field as `True` __only in case when these fields show anything__. It means that they will be omited for cases when they are false which is actually default for these values.

For example, `unlimitedGifts` will have `true` value __only if unlimited regular gifts are accepted__ and will be omited (not set, `false`) if `unlimited regular gifts are not accepted`